README.md

```diff
@@ -21,10 +21,10 @@ at the root directory and you'll get a bundle directory that looks like this:
 ## Watch for changes
 
 ```sh
-yarn watch [firefox | chrome | safari]
+sudo yarn watch [firefox | chrome | safari]
```

Comment:
Why is sudo needed? That makes me a little uneasy

Reply:
sudo is needed, because MV3 requires always to use https to do any request, so in order to solve the certificate issue, the package https-localhost will self-sign a certificate to use and inject it to the system certificate store, in order to be used by the browser. This makes me a little uneasy too, but I think it's good enough for development purposes.

Reply:
small update, i tested self signed certificates without injection and it didn't work.